src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -351,22 +343,37 @@ else if (counter == 2) {
                 StartCord[1]=ycord;
             }
 
-
             //records the starting and ending coordinates for each ship
             if(k==0){
-                this.getShip("submarine").setLocation(new Coordinate(StartCord[0]+1,StartCord[1]+1),new Coordinate(EndCord[0]+1,EndCord[1]+1));
```

Comment:
I think you could get rid of the redundancies in this code by putting the ship names in an array and then iterating through the array with the getShip() function